models/pediatric_abdominal_ct_segmentation/docs/README.md

```diff
@@ -0,0 +1,145 @@
+# Model Overview
+A Pediatric 3D Abdominal Organ Segmentation model, pretrained on adult and pediatric public datasets, and fine tuned for institutional pediatric data.
+
+Please cite this manuscript:
+Somasundaram E, Taylor Z, Alves VV, et al. Deep-Learning Models for Abdominal CT Organ Segmentation in Children: Development and Validation in Internal and Heterogeneous Public Datasets. AJR 2024 May 1 [published online]. Accepted manuscript. doi:10.2214/AJR.24.30931
+
+## Data
+Modality:
+- CT
+
+Organs Segmented:
+- Liver
+- Spleen
+- Pancreas
+
+Pre-training data:
+- Total Segmentator (815)
+- BTCV (30)
+- TCIA Pediatric (282)
+
+Fine-tuning data:
+- Cincinnati Children's Liver Spleen CT dataset (275)
+- Cincinnati Children's Pancreas CT dataset (146)
+
+Testing data:
+- Cincinnati Children's Liver-Spleen (57)
+- Cincinnati Children's Pancreas (35)
+- TCIA-Pediatric (74)
+- Total Segmentator (50)
+
+External dataset licenses can be found in accompanying text file. Internal datasets currently not publicly available.
```

Comment:
could you add more instructions on how to prepare external datasets and train this bundle?

Reply:
More detailed instructions added.